### PR TITLE
Network Autocomplete

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)
Added an autocomplete for the network name field on the custom RPC form.

## Screen recordings / screenshots

http://recordit.co/gvpvlcpFio

### CODE REVIEW

File: src/entries/popup/components/Autocomplete/index.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+export interface AutocompleteProps {
+  data: AutocompleteData;
+  value: string;
+  onChange: (value: string) => void;
+  onSelect: (value: string) => void;
+  onBlur: () => void;
+  onFocus: () => void;
+  borderColor?: string;
```
Comment (greg-schrammel) [inline src/entries/popup/components/Autocomplete/index.tsx:25]:
```suggestion
  borderColor?: BoxStyles['borderColor'];
```

File: src/entries/popup/components/Autocomplete/index.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+              </Command.List>
+            </Box>
+          )}
+        </Command>
+      </Box>
+    );
+  },
+);
+
+Autocomplete.displayName = 'Autocomplete';
```
Comment (greg-schrammel) [inline src/entries/popup/components/Autocomplete/index.tsx:155]: jfyi another option to setting a displayName is using a named function in the forwardRef like
```ts
export const Autocomplete = forwardRef(function Autocomplete(props, ref) {
 ...
})
```

Comment (brunobar79) [reply src/entries/popup/components/Autocomplete/index.tsx:155]: Good to know! That's probably cleaner than using display name. We should probably fix it across the entire codebase later on

File: src/entries/popup/components/Autocomplete/index.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                width: '100%',
+                boxShadow: '2px 12px 12px rgba(0, 0, 0, 0.65)',
+                borderRadius: defaultBorderRadius,
+              }}
+            >
+              <Command.List>
+                {Object.keys(data).map((key) => {
+                  return (
+                    <Command.Group
+                      heading={key}
```
Comment (estebanmino) [inline src/entries/popup/components/Autocomplete/index.tsx:103]: i see the keys of data object are also being used as string for heading here, any option to update it?

**Pull request merged**